src/bolos/fonts_info.c

```diff
@@ -94,6 +94,25 @@ static void parse_nbgl_font(nbgl_font_t *nbgl_font)
   }
 }
 
+// Parse provided NBGL font and add bitmap/character pairs
+static void parse_nbgl_font_12(nbgl_font_t_12 *nbgl_font)
```

Comment:
I don't get why you need to change the font parsing for API_LEVEL_12 and 13 while it was working before?